internal/command/oidc_session.go

```diff
@@ -531,3 +540,26 @@ func tokenReasonToActivityMethodType(r domain.TokenReason) activity.TriggerMetho
 	// all other reasons result in an access token
 	return activity.OIDCAccessToken
 }
+
+func (c *OIDCSessionToken) Encode() (string, error) {
+	data, err := json.Marshal(c)
+	if err != nil {
+		return "", err
+	}
+	return base64.StdEncoding.EncodeToString(data), nil
+}
```

Comment:
No need to base64 encode. Opaque tokens are encrypted and encryption takes bytes.